### DISCUSSION TITLE
Line Chart

### DISCUSSION THREAD

**Discussion by ThucTrieu** [Abhinandan-Kushwaha/react-native-gifted-charts#371] [Q&A] [answered]
Can we add bottom padding for the line chart?
In case the value is 0, the point was cropped
![image](https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/assets/100109060/3563429d-2843-4d18-92f7-3939bd42fe02)

**Comment by ThucTrieu** [comment]
@Abhinandan-Kushwaha  can u help me?

**Comment by Abhinandan-Kushwaha** [comment]
Yes, @ThucTrieu 
This issue will be fixed in the next version of the library which will be released within 2-3 days

**Comment by Abhinandan-Kushwaha** [comment, marked as answer]
Hi @ThucTrieu 👋
This has now been fixed. Please use the latest version of the library.

Now, by default, the chart will have an extra height at the bottom which will be equal to the dataPointsRadius. So, the data points will not be hidden if they coincide with the X axis.

Also, you can add extra height at the bottom using the ` overflowBottom` prop.

**Reply by ThucTrieu** [reply]
many thanks @Abhinandan-Kushwaha
